lab-kyle/server.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
```

Comment:
This is a good start!  Cowsay works with a query and json post to /cowsay which is great.  However, if I send a bad request, it crashes your server.  Your server should recognize a bad request and sent appropriate information, as specified in the assignment.  This issue occurs on both your GET and POST/PUT/DELETE routes.  Your POST/PUT/DELETE routes handle the error but passing the err variable into cowsay causes a fatal error.  The assignment specifies exactly what to return on a 400 error for both routes.  Additionally, the assignment specifies the resource /cowsay and /api/cowsay exist and function congruently.  You'll need to add the /api/cowsay resource.